disperser/apiserver/server.go

```diff
@@ -257,23 +259,18 @@ func (s *DispersalServer) disperseBlob(ctx context.Context, blob *core.Blob, aut
 	if err != nil {
 		for _, param := range securityParams {
 			quorumId := string(param.QuorumID)
-			s.metrics.HandleFailedRequest(quorumId, blobSize, "DisperseBlob")
+			s.metrics.HandleFailedRequest("400", quorumId, blobSize, "DisperseBlob")
 		}
 		return nil, api.NewInvalidArgError(err.Error())
 	}
 
 	if s.ratelimiter != nil {
 		err := s.checkRateLimitsAndAddRates(ctx, blob, origin, authenticatedAddress)
 		if err != nil {
-			for _, param := range securityParams {
```

Comment:
This was wrong accounting: the `checkRateLimitsAndAddRates` will error out and return upon the first error (e.g. first quorum), but this accounting increments metrics for each quorum